data_steward/cdm.py

```diff
@@ -13,6 +13,27 @@
 logger = logging.getLogger(__name__)
 
 
+def create_parser():
```

Comment:
This change and the unit test feels like scope creep.  @ksdkalluri thoughts?

Reply:
@MichaelSchmidt82 Like Lauren said, these changes are good but out of scope. Lets focus on just the scope in 3060 for now, Reason being we may be completely getting rid of this script when we start refactoring registered tier.

Reply:
The function was added for unit testing.  See [here](https://stackoverflow.com/questions/18160078/how-do-you-write-tests-for-the-argparse-portion-of-a-python-module) specifically.  There are other cases where this is done also within the tool directory.  see [here](https://github.com/all-of-us/curation/blob/9741f6364aaca96c87e2d0cf05ce776b710de908/data_steward/tools/delete_stale_test_buckets.py#L85) 

I spoke with @nishanthpp93 about this script, and he suggested a unit test.

Reply:
@MichaelSchmidt82 A unit test is great for testing parsers, so thank you for creating it. However, we don't need it for this particular use case right now, especially for this ticket. Such changes would be good in a separate ticket created for that purpose.

Reply:
Cool.  I'll back it out.